tests/ui/generics/issue-79605.rs

```diff
@@ -0,0 +1,6 @@
+struct X<'a, T>(&'a T);
+
+impl X<'_, _> {}
+        //~^ ERROR the placeholder `_` is not allowed within types on item signatures for implementations
```

Comment:
```suggestion
//~^ ERROR the placeholder `_` is not allowed within types on item signatures for implementations
```
We do not check the column number of the arrow, so unaligned is enough.

Reply:
Fixed.